SwiftLeeds/Data/Model/Contributor.swift

```diff
@@ -0,0 +1,89 @@
+//
+//  Contributor.swift
+//  SwiftLeeds
+//
+//  Created by LUCKY AGARWAL on 20/07/22.
+//
+
+import Foundation
+import UIKit
+
+//Static Filler Data
+
+struct SocialHandle: Hashable{
```

Comment:
Is this needed right now, can't see the list being currently used in the UI?  If it is, I'd expect the type to be an enum so we can handle the correct icons etc when needed

Reply:
Yeah, So it will be used on the About Page. I was unsure how the service's response will be available for this ( when we have the API plugged in ).